docs/testplan/BGP_IPv6_test.md

```diff
@@ -0,0 +1,66 @@
+# SONiC Switch High-Scale IPv6 BGP Test
+
+## Table of Contents
+
+- [Test Objective](#test-objective)
+- [Test Setup](#test-setup)
+- [Test Steps](#test-steps)
+
+## Test Objective
+
+This test verifies the scalability and stability of multiple BGP sessions on a SONiC switch. BGP sessions will be established between each Ethernet logical port of the DUT and its neighboring devices. The test evaluates the DUT’s ability to initiate and maintain BGP sessions, validates proper route learning, and measures BGP update convergence time under various conditions.
+
+## Test Setup
+
+This test is designed to be topology-independent, meaning it does not assume or enforce a specific network layout. The only requirement is that the DUT is fully connected to handle full traffic loads under stress. All logical Ethernet ports are utilized to establish BGP sessions. Assuming the DUT has X logical Ethernet ports and is connected to Y neighboring switches, we will establish X/Y BGP sessions between each neighbor and the DUT.
+
+![Test Setup](./example_layout.png)
+
+In the above example, the DUT has 256 logical Ethernet ports and is connected to 4 neighboring switches, we will establish 64 BGP sessions between each neighbor and the DUT.
+
+## Test Steps
+
+1. Assign a unique AS number to each of the five switches.
```

Comment:
The ASN should be coming from topology. we should mention this in the test setup section as topology introduction.